src/pseudopeople/configuration/interface.py

```diff
@@ -0,0 +1,51 @@
+"""
+===========================
+The Configuration Interface
+===========================
+
+An interface for users to interact with the configuration (config) in pseudopeople.
+
+In ``pseudopeople`` a configuration is used to provide noise level that get passed to the noise functions to apply
+various types of noise to given datasets.  Users can provide their own configuration that will override the default
+values.
+.. code-block:: python
+>>> import pseudopeople as pp
+>>> pp.get_config("decennial_census")
+>>> user_config = {"decennial_census": {"row_noise": {"probability": {"omit_rows": 0.1},},},}
+>>> pp.get_config("decennial_census", user_config)
+
+"""
+
+from pathlib import Path
+from typing import Dict, Union
+
+from loguru import logger
+
+from pseudopeople.configuration.generator import get_configuration
+from pseudopeople.schema_entities import DATASETS
+
+
+def get_config(dataset_name: str = None, user_conf: Union[Path, str, Dict] = None) -> Dict:
+    """
+    Function that displays the configuration for the user
+    :param dataset_name: Name of dataset to lookup in configuration.  Providing this argument returns the configuration
```

Comment:
nit: add a blank line between the description and the params. Also indent multiple line descriptions